src/echo/ws/echo-api.js

```diff
@@ -30,7 +30,10 @@ class EchoApi {
 	 * @returns {Promise}
 	 */
 	exec(method, params) {
-		if (!this.api_id) return Promise.reject(new Error(`Access to ${this.api_name} API does't exist`));
+		if (!this.api_id) {
+			return Promise.reject(new Error(`${this.api_name} API is not available by default,
```

Comment:
@Korblepuskatel , if i will not provide some API, that available by default, and will call its method, then i will see "API is not available by default", that is lie.

Reply:
fixed, I hope so